timed/tracking/migrations/0014_rename_type_absence_absence_type.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.13 on 2022-06-16 13:25
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("tracking", "0013_report_billed"),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name="absence",
+            old_name="type",
+            new_name="absence_type",
+        ),
+    ]
```

Comment:
Field name ```type``` is deprecated since [JSON API version 4.3.0](https://github.com/django-json-api/django-rest-framework-json-api/blob/main/CHANGELOG.md#430---2021-12-10).
In that case I'd have to update our Metabase queries.

Reply:
Fair point. In this case, it's a breaking change, please document it as such ([conventional commits](https://www.conventionalcommits.org/en/v1.0.0/))